datascience/tables.py

```diff
@@ -1003,15 +1004,58 @@ def groups(self, labels, collect=None):
             return grouped
 
     def pivot(self, columns, rows, values=None, collect=None, zero=None):
-        """Generate a table with a column for rows (or a column for each row
-        in rows list) and a column for each unique value in columns. Each row
-        counts/aggregates the values that match both row and column.
-
-        columns -- column label in self
-        rows -- column label or a list of column labels
-        values -- column label in self (or None to produce counts)
-        collect -- aggregation function over values
-        zero -- zero value for non-existent row-column combinations
+        """Generate a table with a column for each unique value in ``columns``,
+        with rows for each unique value in ``rows``. Each row counts/aggregates
+        the values that match both row and column based on ``collect``.
+
+        Args:
+            ``columns`` -- a single column label, (``str``), in self, used to
+                create new columns, based on its unique values in self.
+            ``rows`` -- row labels, as (``str``) or list of strings, used to
+                create new rows based on it's unique values.
+            ``values`` -- column label in self for use in aggregation.
```

Comment:
For these arguments, can we avoid the use of `self`? The students don't know what that means. We should instead say `the table`.